joinwiz_core/src/main/scala/joinwiz/testkit/SeqJoinImpl.scala

```diff
@@ -21,4 +21,11 @@ class SeqJoinImpl[L, R](op: Expr[L, R], left: Seq[L], right: Seq[R]) {
       b <- right
       if op(a, b)
     } yield (a, b)
+
+  def leftAntiJoin(): Seq[L] = {
+    leftJoin()
+      .filter { case (_, right_) => right_.isEmpty }
```

Comment:
you can replace `filter`+`map` with `collect`

Reply:
done